src/core/tracking.js

```diff
@@ -315,4 +317,59 @@ export const sendSponsoredEventBeacon = ({
   });
 };
 
+export const storeConversionInfo = ({
+  _log = log,
+  _window = typeof window !== 'undefined' ? window : {},
+  addonId,
+  data,
+}: {
+  _log?: typeof log,
+  _window?: typeof window | Object,
+  addonId: number,
+  data: string,
+}) => {
+  if (_window.sessionStorage) {
+    _window.sessionStorage.setItem(
+      SPONSORED_INSTALL_CONVERSION_INFO_KEY,
+      JSON.stringify({ addonId, data }),
+    );
+  } else {
+    _log.warn(
+      'window.sessionStorage does not exist. Not storing conversion info.',
+    );
+  }
+};
+
+export const getConversionInfo = ({
+  _window = typeof window !== 'undefined' ? window : {},
+}: {
+  _window?: typeof window | Object,
+} = {}) => {
+  if (_window.sessionStorage) {
+    const info = _window.sessionStorage.getItem(
+      SPONSORED_INSTALL_CONVERSION_INFO_KEY,
+    );
+
+    if (info) {
+      return JSON.parse(info);
```

Comment:
We should probably handle invalid data here. I think `parse` throws.

Reply:
Fixed